omnibus/cookbooks/omnibus-supermarket/recipes/config.rb

```diff
@@ -40,6 +40,25 @@
 # necessary for some community cookbooks.
 node.consume_attributes('runit' => node['supermarket']['runit'])
 
+case node['supermarket']['fips_enabled']
```

Comment:
@cattywampus This case statement to coerce fips_enabled to a boolean value is my latest attempt to address someone putting a string value instead. What do you think?